### PR TITLE
feat: Add Severity List input

### PR DESCRIPTION
**Pull request by pmegremis**
A severity list has been added along with update on read me and
action.yml file.

Contributes: #60

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -133,7 +143,8 @@ const startAnalysis = async () => {
   });
 
   // output the constructed object
-  core.setOutput('nowsecureReportData', reportOutput);
+  // core.setOutput('nowsecureReportData', reportOutput);
```
Comment (sgkiokas) [inline index.js:146]: Please comment this in.

File: index.js [hunk 2]
```diff
@@ -133,7 +143,8 @@ const startAnalysis = async () => {
...
+  console.log(reportOutput);
```
Comment (sgkiokas) [inline index.js:147]: Please remove this comment.

File: README.md [hunk 3]
```diff
@@ -120,6 +121,7 @@ jobs:
                   IOS_PACKAGE: 'ios.mock'
                   ANDROID_PACKAGE: 'android.mock'
                   NOWSECURE_GROUP_ID: ${{ secrets.NOWSECURE_GROUP_ID }}
+                  SEVERITY_LIST: ''
```
Comment (sgkiokas) [inline README.md:124]: Maybe an example like 

```
SEVERITY_LIST: 'medium, want'
```

might be more useful to the end user.

**Review by sgkiokas — CHANGES_REQUESTED** [review]
Some small changes and questions only. Also, could you please create a dummy release and use it to your workflow in order to see that everything flows properly along with the Jira action?

File: index.js (hunk 2 above)
Comment (pmegremis) [reply index.js]: console.log has been removed

File: index.js (hunk 1 above)
Comment (pmegremis) [reply index.js]: comment in core.setoutput

File: README.md (hunk 3 above)
Comment (pmegremis) [reply README.md]: update to SEVERITY_LIST: 'high, medium'

**Comment by sgkiokas** [comment]
Also @pmegremis we should cater for the fact that a user might provide a list of severities with spaces for some reason. So, a `trim` method should be added to the following:

```
if (severityList !== undefined) {
  severityListSplit = severityList.split(',');
}
```

Obviously, this applies to the rest of the inputs that can be set and it was an omission from my side.

**Review by sgkiokas — APPROVED** [review]
LGTM

**Pull request merged**